objects/c4e8cd/script.lua

```diff
@@ -1 +1,977 @@
 difficulty=-1
+postSetup = true
+postSetupComplete = false
+hasBroadcast = true
+
+local destinyPlayers = {}
+local readyTokens = {}
+local round = 1 -- First or second round of drafts
+local phase = 0 -- Phase within a round
+local timer = nil
+
+function onSave()
+    local data_table = {
+        destinyPlayers = destinyPlayers,
+        readyTokens = readyTokens,
+        round = round,
+        phase = phase,
+    }
+    return JSON.encode(data_table)
+end
+
+function onLoad(saved_data)
+    if saved_data ~= "" then
+        local loaded_data = JSON.decode(saved_data)
+        destinyPlayers = loaded_data.destinyPlayers
+        readyTokens = loaded_data.readyTokens
+        round = loaded_data.round
+        phase = loaded_data.phase
+    end
+
+    Color.Add("SoftBlue", Color.new(0.53,0.92,1))
+
+    if Global.getVar("gameStarted") and Global.getVar("scenarioCard") ~= nil and Global.getVar("scenarioCard").guid == self.guid then
+        PostSetup()
+    end
+end
+
+function Broadcast()
+    return "Use the button on the scenario panel to progress drafting"
+end
+
+function PostSetup(_)
```

Comment:
`PostSetup()` takes a `params` argument, which contains data about the adversary. It's thus unnecessary for a scenario. `_` is the convention for taking an argument whose value is ignored, and I believe it's necessary to use that convention in order to avoid the linter complaining about an unused argument.